external_deps/build.sh

```diff
@@ -974,6 +1117,13 @@ build_install() {
 		;;
 	esac
 
+	case "${PLATFORM}" in
+	windows-*-*)
+		# CMake looks for libSDL2.a and aborts if missing if this file exists:
+		rm -rf "${PKG_PREFIX}/lib/cmake/SDL2/SDL2staticTargets.cmake"
```

Comment:
What's this? I don't see a file like this anywhere, and if I revert this commit it does not appear.

Reply:
For me, this file came up and sabotaged the build not for Windows, but for Linux targets by injecting a `-lsamplerate` flag. Anyway on the `slipher/deps-combined` branch I reverted that commit and did a proper fix by configuring the SDL build not to use the dependency.